### ISSUE TITLE
Library failing to build due to missing .proto

### ISSUE THREAD

**Issue by bcoe** [googleapis/nodejs-talent#429]
```
Error: no such Type or Enum 'Interview' in Type .google.cloud.talent.v4beta1.Application
    at Type.lookupTypeOrEnum (/synthtool/node_modules/protobufjs/src/namespace.js:411:15)
    at Field.resolve (/synthtool/node_modules/protobufjs/src/field.js:268:94)
    at Type.set (/synthtool/node_modules/protobufjs/src/type.js:177:38)
    at Type.get (/synthtool/node_modules/protobufjs/src/type.js:155:45)
    at Field.resolve (/synthtool/node_modules/protobufjs/src/field.js:317:21)
    at Type.resolveAll (/synthtool/node_modules/protobufjs/src/type.js:304:21)
    at Namespace.resolveAll (/synthtool/node_modules/protobufjs/src/namespace.js:308:25)
    at Namespace.resolveAll (/synthtool/node_modules/protobufjs/src/namespace.js:308:25)
```

See: https://github.com/googleapis/nodejs-talent/pull/424

**Comment by alexander-fenster** [comment]
There is also a generator issue that is triggered by this API, I'm fixing it in https://github.com/googleapis/gapic-generator-typescript/pull/1145.

**Comment by alexander-fenster** [comment]
I verified that this version can be generated using my fix in the generator v2.14.6, so closing this issue for now and will wait for the regeneration.
